buildSrc/src/main/java/org/elasticsearch/gradle/testfixtures/TestFixturesPlugin.java

```diff
@@ -82,27 +104,54 @@ public void apply(Project project) {
             }
             tasks.withType(getTaskClass("com.carrotsearch.gradle.junit4.RandomizedTestingTask"), task ->
                 extension.fixtures.all(fixtureProject -> {
-                    task.dependsOn(fixtureProject.getTasks().getByName("composeUp"));
-                    task.finalizedBy(fixtureProject.getTasks().getByName("composeDown"));
-                    // Configure ports for the tests as system properties.
-                    // We only know these at execution time so we need to do it in doFirst
-                    task.doFirst(it ->
-                        fixtureProject.getExtensions().getByType(ComposeExtension.class).getServicesInfos()
-                            .forEach((service, infos) ->
-                                infos.getPorts()
-                                    .forEach((container, host) -> setSystemProperty(
-                                        it,
-                                        "test.fixtures." + fixtureProject.getName() + "." + service + "." + container,
-                                        host
-                                    ))
-                            ));
-                }));
+                    fixtureProject.getTasks().matching(it->it.getName().equals("buildFixture")).all(buildFixture ->
+                        task.dependsOn(buildFixture)
+                    );
+                    fixtureProject.getTasks().matching(it->it.getName().equals("composeDown")).all(composeDown ->
+                        task.finalizedBy(composeDown)
+                    );
+                    configureServiceInforForTask(
+                        task,
+                        fixtureProject,
+                        (name, port) -> setSystemProperty(task, name, port)
+                    );
+                })
+            );
         }
     }
 
+    private void configureServiceInforForTask(Task task, Project fixtureProject, BiConsumer<String, Integer> consumer) {
+        // Configure ports for the tests as system properties.
+        // We only know these at execution time so we need to do it in doFirst
+        task.doFirst(theTask ->
+            fixtureProject.getExtensions().getByType(ComposeExtension.class).getServicesInfos()
+                .forEach((service, infos) -> {
+                    theTask.getLogger().info(
+                        "Port maps for {}\nTCP:{}\nUDP:{}\nexposed to {}",
```

Comment:
Info seems a bit noisy for this.

Reply:
Info seems right to me. The default logging level is lifecycle, so you will not see this by default, but only when using `--info` (which is what we use in CI).